src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1939,7 +1939,8 @@
     </Field>
   </Node>
   <Node Name="BinaryPatternSyntax" Base="PatternSyntax">
```

Comment:
You probably want to remove `==` and `!=` from relational patterns too (a few lines above this).
The parser still accept those, perhaps good for error recovery?

Reply:
Yes, present for error recovery

---
In reply to: [386532742](https://github.com/dotnet/roslyn/pull/42023#discussion_r386532742) [](ancestors = 386532742)